api/server/server.go

```diff
@@ -847,12 +847,28 @@ func postContainersRestart(eng *engine.Engine, version version.Version, w http.R
 	if vars == nil {
 		return fmt.Errorf("Missing parameter")
 	}
-	job := eng.Job("restart", vars["name"])
-	job.Setenv("t", r.Form.Get("t"))
-	if err := job.Run(); err != nil {
+
+	seconds := 10
+	t := r.Form.Get("t")
+	if t != "" {
+		s, err := strconv.Atoi(t)
+		if err != nil {
+			return err
+		}
+		seconds = s
+	}
+	//s, err := strconv.Atoi(t)
```

Comment:
I'm not sure if the code above is ok because in `daemon.ContainerRestart` there was a default 10 seconds to `container.Restart` only if not `job.EnvExists("t")` but cli always set 10 seconds default from cli/restart. Maybe old api?